utils/capellacontrol/controller.go

```diff
@@ -1107,3 +1107,27 @@ func (c *Controller) GetTrustedCAs(
 
 	return resp, err
 }
+
+type CreateUpdateServerVersionRequest struct {
+	OverrideToken string `json:"token"`
+	ServerImage   string `json:"image"`
+	ServerVersion string `json:"server"`
+}
+
+func (c *Controller) UpdateServerVersion(
+	ctx context.Context,
+	tenantID, projectID, clusterID string,
+	req *CreateUpdateServerVersionRequest,
+) error {
+	c.logger.Info(req.ServerImage)
```

Comment:
Generally we shouldn't need to log these details since they get logged at a higher level during execution of these operations.  Additionally, if there is some specific value to logging these things, we should provide context for what it is that is being logged.  We also are using zap logging which is a structured logger, so the server image should be passed as a parameter to that, as opposed to being the message itself (there are lots of examples elsewhere in this file), for instance:
```
c.logger.Info("updating server version",
  zap.String("image", req.ServerImage),
  zap.String("version", req.ServerVersion))
```

Reply:
okay